tests/helpers/destroy-app.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember';
+import { cleanup } from 'ember-wait-for-test-helper/wait-for';
 
 export default function destroyApp(application) {
+  cleanup();
```

Comment:
Not necessary, it makes sure no waiters are left running at the end of each test. If you want automatic cleanup you need this line, but if all your waiters resolve successfully then this should be a no-op.